lib/improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -821,6 +819,21 @@ def __init__(
             except CoordinateNotFoundError:
                 pass
 
+        # Check that the domain of the current forecast and coefficients cube
+        # matches.
+        for axis in ["x", "y"]:
+            current_forecast_points = [
+                current_forecast.coord(axis=axis).points[0],
+                current_forecast.coord(axis=axis).points[-1]]
+            if not np.allclose(current_forecast_points,
+                               coefficients_cube.coord(axis=axis).bounds):
+                msg = ("The domain along the {} axis given by the "
+                       "current forecast {} does not match the domain given "
+                       "by the coefficients cube {}.".format(
+                        axis, current_forecast_points,
+                        coefficients_cube.coord(axis=axis).bounds))
+                raise ValueError(msg)
+
```

Comment:
Accidentally added from other PR? #895

Reply:
Yes, I definitely did not intend to add this code block.